packages/backend/migrations/20230718142550_create_combined_payments_view.js

```diff
@@ -0,0 +1,41 @@
+/**
+ * @param { import("knex").Knex } knex
+ * @returns { Promise<void> }
+ */
+/**
+ * @param { import("knex").Knex } knex
+ * @returns { Promise<void> }
+ */
+exports.up = function (knex) {
+  return knex.schema.createViewOrReplace(
+    'combinedPaymentsView',
+    function (view) {
+      const sharedColumns = [
+        'id',
+        'paymentPointerId',
+        'state',
+        'client',
+        'createdAt',
+        'updatedAt',
+        'metadata'
+      ]
+      view.columns([...sharedColumns, 'type'])
+      view.as(
+        knex('incomingPayments')
+          .select([...sharedColumns, knex.raw("'INCOMING' AS type")])
+          .unionAll((union) => {
+            union
+              .select([...sharedColumns, knex.raw("'OUTGOING' AS type")])
+              .from('outgoingPayments')
+          })
+      )
+    }
+  )
+}
```

Comment:
I debated about the best way to combine the payments in this comment https://github.com/interledger/rafiki/issues/1502#issuecomment-1638438973.

Ultimately I went with the union all view because it's the simplest, cannot accidentally be written to, and didn't have the performance concerns I originally thought it might. I was concerned it would perform a union of all the records but with pagination it does not. Confirmed with some against a lot of dummy data.